src/test/java/tests/SelectionPlaylistTest.java

```diff
@@ -0,0 +1,189 @@
+package tests;
+
+import Page_Objects.TestBase;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class SelectionPlaylistTest {
+
+    private ChromeDriver driver;
+    private TestBase testBase;
+    String compSongID = "xavier-rudd-stoney-creek";
+    String almSongID = "electric-light-orchestra-bluebird";

```

Comment:
I think it's better to just use single song that is not on selection playlist - smaller chance that any of the tests fail